data-transform.qmd

```diff
@@ -678,8 +697,7 @@ As you can see, when you summarize an ungrouped data frame, you get a single row
     )
     ```
 
-    a.  What does the following code do?
-        Run it, analyze the result, and describe what `group_by()` does.
+    a.  Write down what you think the output will look like, then check if you were correct, and describe what `group_by()` does.
```

Comment:
Make this an unordered list to match the exercises you changed above?

Reply:
These are parts (a), (b), (c), etc., should be completed consecutively, vs. the other one was filtering for all of those conditions at once (or at least I thought so). If not, I should bring the other one back.